.circleci/config.yml

```
@@ -67,5 +122,10 @@ workflows:
           context:
             - static-analysis
           name: "Scan with Semgrep"
+  "Malware Scanner":
+    jobs:
+      - reversing-labs:
+          context:
+            - okta-dcp 
```

```suggestion
            - static-analysis 
```

This change was proposed by @bretterer earlier today.